internal/config/config.go

```diff
@@ -166,6 +167,42 @@ func registerFlags() {
 	fs.StringArray(
 		PrometheusTargetsKey, []string{}, "The target URI(s) of Prometheus endpoint(s) for metrics collection.",
 	)
+	fs.String(
+		CommandServerHostKey, "", "The target hostanme of command server endpoint for command and control.",
```

Comment:
Also, maybe save the default values in `defaults.go`?

Reply:
Looks like the other descriptions have also omitted the "the" before "command server endpoint". I feel it would be more grammatically correct to have it but okay with leaving it out too.